### ISSUE TITLE
Support for hardware without file systems

### ISSUE THREAD

**Issue by Coderitter-GmbH** [alrevuelta/cONNXr#77]
There is already an example how to run a model, without a file system. My suggestion is, that with an additional flag, the file system supporting functions can be disabled, which makes porting easier, because there is no need to mock it.

**Comment by alrevuelta** [comment]
Fair request. I kind of ran into this when trying to use cONNXr [with Arduino](https://github.com/alrevuelta/cONNXr/blob/master/examples/arduino_example/Arduino_Example_ONNX/Arduino_Example_ONNX.ino) Is that the example you are referring to?

**Comment by Coderitter-GmbH** [comment]
No, i used https://github.com/alrevuelta/cONNXr/blob/master/examples/example2/example2.c
